.php_cs

```diff
@@ -38,46 +38,47 @@ $config->setRules(
     [
         '@PSR2' => true,
         'align_multiline_comment' => ['comment_type' => 'all_multiline'],
+        'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'blank_line_before_statement' => [
             'statements' => ['return', 'continue', 'break', 'declare', 'throw', 'try'],
         ],
-        'cast_spaces' => ['space' => 'none'],
-        'combine_consecutive_issets' => true,
-        'combine_consecutive_unsets' => true,
+        'binary_operator_spaces' => true,
```

Comment:
👎

Reply:
Despite its name, rule can be used to align pretty much [all operators](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/53677f035c3c4ebd8d10d95c71f4a15cc6610df4/src/Fixer/Operator/BinaryOperatorSpacesFixer.php#L88-L133), which IMO should be surrounded by a single space.

e.g.

```php
// before
$delta  =$b**2-4 *$a* $c;
// after
$delta = $b ** 2 - 4 * $a * $c;
```